fenics/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - mshr =2019.1.0
+  - matplotlib
```

Comment:
```suggestion
```
I think `matplotlib` is already included in the jupyter notebook container, so it is not necessary to add it here.

Reply:
Done!